### PR TITLE
Documentation for mod_extdisco XEP-0215

### PR DESCRIPTION
**Pull request by janciesla8818**
This PR adds documentation for mod_extdisco that is implementing XEP-0215. #2870

### CODE REVIEW

**Comment by mongoose-im** [comment]
[8552.1](https://travis-ci.org/esl/MongooseIM/jobs/727408112) / Erlang 23.0.3 / small_tests / 12d86fc06c6550cb6192d17ed55c8e5223bb5920
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2871/8552/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2871/8552/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-09-15_15.24.47/lib.mongooseim.logs/run.2020-09-15_15.24.47/suite.log.html)

**Review by NelsonVides — COMMENTED** [review]
Seems cool, though as you can imagine I'd merge this one only after the implementation is merged.

Just one question, the document says that certain parameters are required, like host and port, but this isn't enforced anywhere in the code (I think, right?). I'm not sure if we plan to do so in the validation code of the new config, but perhaps the module itself could already check? I don't really have an opinion here, just asking outloud :)

File: doc/modules/mod_extdisco.md
Comment (aleklisi) [inline doc/modules/mod_extdisco.md:8]: See example 2 from https://xmpp.org/extensions/xep-0215.html#all
the last service is actually of type `ftp` 🤔

File: doc/modules/mod_extdisco.md
Comment (aleklisi) [inline doc/modules/mod_extdisco.md:12]: The `port` option is recommended but not required AFAIK https://xmpp.org/extensions/xep-0215.html#table-1

File: doc/modules/mod_extdisco.md
Comment (aleklisi) [inline doc/modules/mod_extdisco.md:11]: If it is a required option why fo you provide a default value? Let it crush

**Comment by aleklisi** [comment]
Docs generally look ok, I might have some more questions to the #2870 .

File: doc/modules/mod_extdisco.md
Comment (janciesla8818) [reply doc/modules/mod_extdisco.md:11]: The default value is not set.

**Comment by mongoose-im** [comment]
[8557.1](https://travis-ci.org/esl/MongooseIM/jobs/728013941) / Erlang 23.0.3 / small_tests / 55394c2a827afdc08cb7b5c741035bfa279622f5
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2871/8557/small_tests.23.0.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2871/8557/small_tests.23.0.3/small/ct_run.mongooseim@localhost.2020-09-17_13.53.33/lib.mongooseim.logs/run.2020-09-17_13.53.33/suite.log.html)

**Review by NelsonVides — APPROVED** [review]

**Pull request merged**
